packages/react/src/Button/Button.features.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import {EyeIcon, TriangleDownIcon, HeartIcon} from '@primer/octicons-react'
 import React, {useState} from 'react'
-import {Button} from '.'
+import {Button, Link} from '.'
```

Comment:
Would it be possible to import `Link` from `'../Link'`? This could help to prevent the circular reference in the `Button/index.js` entrypoint that's happening 👀